src/components/ComponentList.js

```diff
@@ -175,6 +178,7 @@ export default class ComponentList extends React.Component {
     let expressions = []
     let declared = []
 
+    console.log("Facets: " + JSON.stringify(facets))
```

Comment:
hmmm, let's remove this

Reply:
yeah, committed without cleaning up my debugging